en/core-libraries/time.rst

```diff
@@ -180,6 +180,11 @@ The following calendar types are supported:
 
 .. versionadded:: 3.1
     Non-gregorian calendar support was added in 3.1
+    
+.. note::
+    For constant strings (ie IntlDateFormatter::FULL) Intl uses ICU library 
+    that feeds its data from CLDR (http://cldr.unicode.org/) which version 
```

Comment:
not sure how this link would help clarify what is CLDR, IMHO it only makes user harder to find that info